### PR TITLE
Use tighter except blocks

### PR DESCRIPTION
**Pull request by merwok**

### CODE REVIEW

File: taskipy/run.py [hunk 1]
```diff
@@ -20,36 +20,38 @@ def run_commands_and_bail_on_first_fail(cmds: List[str]) -> int:
...
 
     try:
         task = tasks[task_name]
-        command_with_passed_args = ' '.join([task] + args)
-        commands.append(command_with_passed_args)
-    except:
+    except KeyError:
         print(f'could not find task "{task_name}""')
         sys.exit(127)
+
```
Comment (merwok) [inline taskipy/run.py]:
```suggestion

```

File: taskipy/run.py [hunk 2]
```diff
@@ -20,36 +20,38 @@ def run_commands_and_bail_on_first_fail(cmds: List[str]) -> int:
     except FileNotFoundError:
         print('no pyproject.toml file found in this directory')
         sys.exit(1)
-    except:
+    except Exception:  # pylint: disable=W0703
```
Comment (illBeRoy) [inline taskipy/run.py]: If we follow the the pattern we've set with `except KeyError`, how about we intercept `toml.TomlDecodeError` here? wdyt?

Comment (merwok) [reply taskipy/run.py]: I didn’t know which exception to catch!

**Comment by illBeRoy** [comment]
@all-contributors please add @merwok for code

**Comment by illBeRoy** [comment]
Your contribution is now part of [v.1.1.1](https://pypi.org/project/taskipy/)

**Pull request merged**